### PR TITLE
feat: optimize ram

### LINKED ISSUE #80: Check Balance to optimize RAM
Issue reported by sergio in order to optimize RAM 

Add action in eosjs as

Sometimes users won't have the token for RAM payments

### PR DESCRIPTION
**Pull request by adriexnet**
### GH Issue
Resolve #80 

### What does this PR do?
optimize ram using open action of token 2 so that the user pays for the RAM

### Steps to test
1. yarn start
2. navigate to http://localhost:3000/exchange
3. exchange using a token that you do not have in your account
4. confirm that there is a action open in the details of the transaction 

#### CheckList
- [x] Follow proper Markdown format
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

File: src/language/es/es.json [hunk 1]
```diff
@@ -99,10 +99,11 @@
     "pairSupply": "Circulante",
     "estimatedPrice": "Precio",
     "btnLabel": "Operar",
-    "price": "Precio"
+    "price": "Precio",
+    "yourWallet": "Su billetera"
```
Comment (xavier506) [inline src/language/es/es.json:103]: Should there be a capital B? in `billetera` ?

Comment (adriexnet) [reply src/language/es/es.json]: I think so, I update it in a new commit 👍

**Review by xavier506 — APPROVED** [review]
LGTM

File: src/utils/evolutiondex.js [hunk 2]
```diff
@@ -203,19 +203,53 @@ const getExchangeAssetsFromToken2 = (amount, pair) => {
...
+    const authorization = [
+      {
+        actor: ual.activeUser.accountName,
+        permission: 'active'
+      }
+    ]
+    let aditionalActions = []
+    const balance = await getUserTokenBalance(ual, pair.pool2)
+
+    if (!balance) {
```
Comment (xavier506) [inline src/utils/evolutiondex.js:231]: @sergioyuhjtman  please review

File: src/components/InputTextAndSelect/InputTextAndSelect.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import React, { useState, useEffect, useRef } from 'react'
+import clsx from 'clsx'
+
```
Comment (tetogomez) [inline src/components/InputTextAndSelect/InputTextAndSelect.js:3]: delete this blank space

**Review by tetogomez — APPROVED** [review]
LGTM

Just delete the blank space on imports

File: src/components/InputTextAndSelect/InputTextAndSelect.js (hunk 3 above)
Comment (adriexnet) [reply src/components/InputTextAndSelect/InputTextAndSelect.js]: nice catch, thanks

**Pull request merged**